torch/csrc/distributed/rpc/init.cpp

```diff
@@ -444,6 +445,48 @@ If the future completes with an error, an exception is thrown.
           &ProcessGroupAgent::sync,
           py::call_guard<py::gil_scoped_release>());
 
+  // Base class: torch.distributed.rpc.RpcBackendOptions.
+  py::class_<TensorPipeRpcBackendOptions>(
```

Comment:
(Not request changes in this PR). We will need API docs for this type. Added #37783 to track.